src/pages/RealtimeMapPage.tsx

```diff
@@ -138,34 +142,46 @@ export default function RealtimeMapPage() {
   console.log(paths)
 
   return (
-    <div className="map-container">
-      <div className="map-header">
-        <h1>Realtime Map</h1>
-        <div className="map-header-buttons">
-          <label>
-            {TEXTS.from_date}{' '}
-            <DataAndTimeSelector
-              timestamp={moment(from.slice(0, 16))} // remove timezone and seconds
-              setTimestamp={(ts) => {
-                const value = ts ? ts.format() : ''
-                setFrom(value)
-                setTo(formatTime(+new Date(value) + (+new Date(to) - +new Date(from)))) // keep the same time difference
-              }}
-              showTimePicker={true}
-            />
-          </label>{' '}
-        </div>
-        <div className="map-header-buttons">
-          <label>
-            {TEXTS.watch_locations_in_range}{' '}
-            <MinuteSelector
-              num={(+new Date(to) - +new Date(from)) / 1000 / 60}
-              setNum={(num) => setTo(formatTime(+new Date(from) + +num * 1000 * 60))}
-            />{' '}
-            {TEXTS.minutes}
-          </label>
-        </div>
-        <div className="map-header-buttons">
+    <PageContainer className="map-container">
+      <Grid container spacing={2} sx={{ maxWidth: INPUT_SIZE }}>
+        <Grid xs={12}>
+          <Label text={TEXTS.realtime_map_explanation} />
+        </Grid>
+        {/* from date */}
+        <Grid xs={2}>
```

Comment:
You should use here the "GridSelectorAndLabel" component

Reply:
The layout here needs to be a little bit different from "GridSelectorAndLabel".
The 'xs' for the label need be 2 and the 'xs' for the content need be 10.
To fix it I have two methods.

method 1:
move xsLabel and xsContent  by property.
method 2:
not use  "GridSelectorAndLabel".

Which method to choose?

Reply:
In my opinion, the first method is better so you could use this component in different places as well. Let's hear from others to see what they think about it :) @NoamGaash @ShayAdler

Reply:
I remove the `GridSelectorAndLabel` completely, 
Grid containers and items better to be in the same component.